sources/c_api/compression_operations/qpl_deflate_slow.c

```diff
@@ -129,8 +129,8 @@ static inline deflate_match_t own_get_best_match(const deflate_hash_table_t *con
 #endif
     uint32_t        current_number_of_attempts = hash_table_ptr->attempts;
     deflate_match_t best_match                 = {
-            .index = (uint32_t) (current_match_ptr - lower_bound_ptr),
             .length = match_length,
+            .index = (uint32_t) (current_match_ptr - lower_bound_ptr),
```

Comment:
This fixes an out-of-order declaration for the fields of the struct. It was exposed by moving this translation unit to a C++ compiler. The change was generated automatically using a large language model.